### PR TITLE
update the doc to include description for the concurrent worker limit config for azure-openai integraction

### PR DESCRIPTION
**Pull request by grant-Kolena**
### Please check if the PR fulfills these requirements

- [ ] Include reference to internal ticket and/or GitHub issue "Fixes #NNNN" (if applicable)
- [ ] Relevant tests for the changes have been added
- [ ] Relevant docs have been added / updated

### CODE REVIEW

File: docs/connecting-external-llm-endpoint/azure-openai.md [hunk 1]
```diff
@@ -47,12 +47,13 @@ We will assign a role to the App registration created above:
...
+| Client ID               | The Application (client) ID of the App registration created in [step 1](#step-1-create-azure-llm-registration-for-kolena)         |
+| Client Secret           | The secret key for the App registration created in [step 1](#step-1-create-azure-llm-registration-for-kolena)                     |
+| Endpoint                | The Azure OpenAI endpoint, it can be find under `Resource Management/Keys and Endpoint` section in Azure OpenAI console           |
+| Model Name              | The Name of the model to be displayed on kolena                                                                                   |
+| Deployment Name         | The deployment name for the model to be deployed on Kolena Resource `Management/Model Deployment` section in Azure OpenAI console |
+| Support Image           | Whether this hosted model has vision capabilities                                                                                 |
+| Concurrent Worker Limit | The number of concurrent API request we can make to your hosted LLM endpoint per Prompt Extraction Pipeline run                   |
```
Comment (nankolena) [inline docs/connecting-external-llm-endpoint/azure-openai.md]: nit - `concurrent API requests` plural

Comment (nankolena) [reply docs/connecting-external-llm-endpoint/azure-openai.md]: same in the top, `50 concurrent requests`

**Review by nankolena — APPROVED** [review]
approving to unblock assuming nits will be addressed

File: docs/connecting-external-llm-endpoint/azure-openai.md (hunk 1 above)
Comment (grant-Kolena) [reply docs/connecting-external-llm-endpoint/azure-openai.md]: will make the change

**Pull request merged**